_includes/footer.html

```diff
@@ -0,0 +1,37 @@
+<footer>
+    <div class="footer" role="navigation">
+        <div class="container">
+            <div class="navbar-text">
+                <ul class="footer-text">
+                    <li><a href="{{site.baseurl}}/index.html">Home</a></li>
```

Comment:
We should comment out this line since `Home` is included in [nav.yml](https://github.com/sjsrey/pysal.github.io/blob/jekyll/_data/nav.yml#L2). Now we have two `Home` in the footer.